Demo/src/main/res/layout/pay_later_button_test_layout.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- XML view for testing only. These aren't used anywhere else in the demo application. -->
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:ignore="UnusedResources"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <com.paypal.android.paymentbuttons.PayLaterButton
+        android:id="@+id/button1"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="8dp"
+        android:padding="8dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:paylater_color="white"
+        />
```

Comment:
```suggestion
        app:paylater_color="white" />
```

Nit - I'm not sure if we have any custom styling for XML files, but I think the default formatting doesn't have the closing tag on a new line.

Reply:
🤷 I do it because when I want to delete the last line of text, I can do a `cmd+del` and that would still leave the closing tag intact.
I'm trying to move away from thinking too much about formatting, so if you feel strongly about the tag on the newline, I'll be happy to change it back.

Reply:
I see! That reasoning makes a lot of sense. I'm just worried about folks who will auto format the entire file. Should we make a modification to the format preference so Android Studio picks up this change?

Should we also follow this pattern for XML tags that have separate open and close tags?

Reply:
All good questions. I don't have a preference, tbh.
I've auto-formatted the whole file, and now all the tags are on the same line as the last one.

If there's a common setting (not on individual IDEs) that can take care of this formatting for us, I'm up for it.

Reply:
<img width="535" alt="Screenshot 2024-05-22 at 9 37 01 AM" src="https://github.com/paypal/paypal-android/assets/5005216/12e99d64-5fee-4fd1-8054-c2f001c880d9">

I think enabling this setting in Android Studio will have auto format add that extra new line. We could check in a format file, but everyone would need to manually apply the file in their IDE 🙁

Reply:
I've changed it to the regular expected formatting. We can look at this later. Not an important one :)

Reply:
Good to know about the setting though. :)